### ISSUE TITLE
Training/testing in Keras

### ISSUE THREAD

**Issue by jan-christiansen** [sagr4019/ResearchProject#62]
Check how the training and testing of the examples "Finding and correcting syntax errors" and "Sequence Classification with LSTM" works. For example, is a fixed percentage of data used for testing?

**Comment by Salvos** [comment]
### Finding and correcting syntax errors
The authors of this publication, took the top 10.000 open-source JavaScript projects from Github (594.681 JavaScript files). The filtered the syntactically-valid JavaScript files (494.352 left),tokenized them with Esprima 3.1.1 and removed minified files (479.998 left).

To prepare the neuronal network, the authors created five identically partitions which consists of 
- a training set of ~19M tokens
- a validation set of ~3M tokens
- a test set of 128 files

### Sequence Classification with LSTM
In this publication, the authors used the top 5.000 words from an IMDB dataset and splitted these into

- a training set with 2.500 words (**50%**)
- and a test set with 2.500 words (**50%**)
